articles/cosmos-db/sql/sql-query-is-number.md

```diff
@@ -37,17 +37,18 @@ IS_NUMBER(<expr>)
 SELECT   
     IS_NUMBER(true) AS isNum1,   
     IS_NUMBER(1) AS isNum2,  
```

Comment:
```suggestion
    IS_NUMBER(true) AS isBooleanANumber,   
    IS_NUMBER(1) AS isNumberANumber, 
```

I think there's value in upgrading to a new naming scheme here. I know this was not the original intention of your PR, but let's make this example a little clearer.